blog/good-compliance-programs-do-not-require-fancy-platforms.md

```diff
@@ -0,0 +1,235 @@
+And they may not be able to help accurately translate the expectations of your auditing partner into action steps.
+
+Also, something that all compliance platforms suffer from is that using an additional third party system means that it is yet another external system to login to and learn how to use.
+Think about it.
+Managing Control Statements and evidence outside of normal business tools, often means "out of sight, out of mind" and greatly reduces visibility into compliances processes internally.
+
+If you can afford a personalized consultant, such as Martin Cozzi, I would highly recommend that route.
+
+Consultants like Martin take the time to understand your business and the expectations of the auditing firm to craft a compliance strategy that is best suited to your company.
+This also means that they can translate auditor expectations into action steps for you.
+They can help write custom Control Statements that accurately reflect what you are doing and map them to compliance criteria.
+And my favorite part, consultants like Martin can show you how easy it is to manage compliance materials using the project management tools you **already use**.
+And if you don't use project management tools, then all you need is a Google spreadsheet.
+This is important because it shows once you are oriented in the compliance world, you don't need a fancy UI and tools to keep track of the state of your compliance efforts.
+
+### 2. Do what you say you are doing
+
+This is the next most important thing to do.
+When you write a Control Statement, it must be one that is accurate and true.
+And you want your implementation of that Control to be done in a way where you can prove that you did those things during a period of time.
+
+For example, if you have a Control Statement that says "The engineering team is notified whenever there is a release to production", then (A) that should absolutely be happening, and (B) you should be able to prove that.
+
+But really think about this statement for moment, for it forces you to define the following things:
+- What is the "engineering team", who is in that scope?
+- What is our method/mechanism of notification?
+- What is considered a "release to production"?
+
+By ironing all of that out, and ensuring that there is a history of evidence to gather, this Control Statement, and the actions you take that are associated with it, can scale with the company for a very long time.
+
+### 3. Internally manage Control Statements and resources
+
+This one piggybacks a bit on the first key to success.
+And it does draw a line in the sand for me when deciding whether to use a compliance platform.
+But I feel strongly about it because I have seen just how much more engagement there is across the company when Control Statements and resources are managed using "normal" and "routine" tools.
```

Comment:
```suggestion
I feel strongly about it because I have seen just how much more engagement there is across the company when Control Statements and resources are managed using "normal" and "routine" tools.
```